dspace-api/src/main/java/org/dspace/util/SolrImportExport.java

```diff
@@ -298,8 +299,11 @@ private static void reindex(String indexName, String exportDirName, boolean keep
             }
 
             // Create a temp directory to store temporary core data
-            File tempDataDir = new File(configurationService.getProperty(
-                "dspace.dir") + File.separator + "temp" + File.separator + "solr-data");
+            File tempDataDir = new File(configurationService.getProperty("solr.home") + File.separator + "server"
+                               + File.separator + "solr"
+                               + File.separator + "configsets" + File.separator + "statistics"
+                               + File.separator + "data"
+                               + File.separator + "data");
```

Comment:
There appears to be two `data` folders here? Was that intentional?  It looks like this translates to `/statistics/data/data` which seems like a confusing directory structure.